src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -616,6 +616,12 @@ public T execute(final String merchantAccount, final PaymentData paymentData, fi
         }
     }
 
+    private final Map<String, Object> propertiesToMapWithPropertyFiltering(final Iterable<PluginProperty> properties, TenantContext context) {
+        final Map<String, Object> map = PluginProperties.toMap(properties);
+        map.entrySet().removeIf(entry -> this.getConfigProperties(context).getSensitivePropertyKeys().contains(entry.getKey().toLowerCase()));
```

Comment:
We don't like lambdas 😬 Could you switch to the old-school™ anonymous syntax?

Reply:
Changed in a7a7d82